### DISCUSSION TITLE
无法购买

### DISCUSSION THREAD

**Discussion by hyz970703** [getlantern/lantern-forum-cn#953] [Q&A]
Mike，帮忙看看是什么情况，现在使用的是安卓端，无法购买续费，重新安装后要么提示用户ID为整数，或者一直连不上，邮箱394171452@qq.com![Screenshot_20241011_020211.jpg](https://github.com/user-attachments/assets/faff4930-666b-4ca8-aace-125d0769933e)

![Screenshot_20241011_014853.jpg](https://github.com/user-attachments/assets/00468e8e-7c7f-4bb0-9410-f01aacb71136)

**Comment by hyz970703** [comment]
> Mike，帮忙看看是什么情况，现在使用的是安卓端，无法购买续费，重新安装后要么提示用户ID为整数，或者一直连不上，邮箱394171452@qq.com

![Screenshot_20241011_020211.jpg](https://github.com/user-attachments/assets/9e4ff9cc-5e3c-40ac-aefe-8d9b26b6b966)

    @Mike777Khoo Mike帮忙看看我这边的情况需要怎么弄😁

**Comment by MaxMKMax** [comment]
@hyz970703 您好. 好的, 我帮您提交问题上去. 谢谢您通知我们. 我让支付部门查看是什么问题.

**Comment by MaxMKMax** [comment]
@hyz970703 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by hyz970703** [reply]
目前无法使用支付宝购买，提示Android err_cleartext_permitted，是否是因为安卓9不支持未加密的流量，添加了新的限制导致的？然后lantern反馈界面也是无法使用报错截图如下![Screenshot_20241012_181951.jpg](https://github.com/user-attachments/assets/f07b956a-dfe1-424b-8fb4-9631023aeaff)

**Comment by greySkiesMarsh** [comment]
Hi @hyz970703 I've upgraded your account to Pro for one month. Please try this link to complete your purchase https://buy.stripe.com/8wM00A5sf15A0yk3cf

**Reply by hyz970703** [reply]
could i purchase with Alipay？beacause i forget cvc
